app/src/main/java/lt/vilnius/tvarkau/fragments/MyProfileFragment.java

```diff
@@ -30,12 +34,37 @@ public static MyProfileFragment getInstance() {
     @Bind(R.id.profile_telephone)
     EditText mProfileTelephone;
 
+    @Bind(R.id.profile_picture)
+    ImageView mProfilePicture;
+
     @Nullable
     @Override
     public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
         View view = inflater.inflate(R.layout.my_profile, container, false);
 
         ButterKnife.bind(this, view);
+
         return view;
     }
+
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        setUpUserProfile();
+    }
+
+    private void setUpUserProfile(){
+        Profile profile;
+
+        if(getActivity()!=null){
```

Comment:
Formatting issue related to spaces.
```java
if(getActivity( )!= null) {
```
 Also can you give a scenario when this if will fail ?